test/unit/peg/PriceBoundPegStabilityModule.test.ts

```diff
@@ -1056,74 +1066,6 @@ describe('PriceBoundPegStabilityModule', function () {
         const amount = 10_000_000;
         await asset.mint(psm.address, amount);
         await psm.connect(impersonatedSigners[pcvControllerAddress]).withdraw(userAddress, await psm.balance());
-
-        const endingBalance = await psm.balance();
-        expect(endingBalance).to.be.equal(0);
-        expect(await asset.balanceOf(userAddress)).to.be.equal(amount);
-      });
-    });
-  });
-
-  describe('PCV', function () {
```

Comment:
The primary way I got speedups in this test file were by changing beforeEach to before; since these pcv tests are very dependent on having independent state, it broke this section. This is intended as a temp fix as we're porting over to forge, however, so I think that's probably okay. Can redo this section if we think I should.